SwiftyTesseract/SwiftyTesseract/Enums/ResultIteratorLevel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TesseractResultIteratorLevel.swift
+//  SwiftyTesseract
+//
+//  Created by Antonio Zaitoun on 17/04/2020.
+//  Copyright © 2020 Steven Sherry. All rights reserved.
+//
+
+import Foundation
+import libtesseract
+
+/*
+ RIL_BLOCK,
+  RIL_PARA,
+  RIL_TEXTLINE,
+  RIL_WORD,
+  RIL_SYMBOL
+ */
+public enum ResultIteratorLevel: Int{
+    public typealias RawValue = Int
```

Comment:
This doesn't need to be defined here since it's defined in the enum declaration.